ingest/src/sources/mongodb-university/MongoDbUniversityDataSource.test.ts

```diff
@@ -0,0 +1,69 @@
+import "dotenv/config";
+import {
+  filterOnlyPublicActiveTiCatalogItems,
+  makeMongoDbUniversityDataSource,
+} from "./MongoDbUniversityDataSource";
+jest.setTimeout(100000);
+
+const baseUrl = "https://api.learn.mongodb.com/rest/catalog";
```

Comment:
was intentional, but not necessary here (see [this comment](https://github.com/mongodb/chatbot/pull/247/#discussion_r1409850535)).

do you think there's a strong reason to use staging instead of prod here?

Reply:
I see, just wanted to check if this was intended. Given your earlier comment regarding consistency with other data APIs, I think this is fine. It doesn't look like there would be that many requests from CI jobs anyhow.